tests/rptest/services/polaris_catalog.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+import json
+import collections
+import re
+from typing import Optional, Any
+
+from ducktape.services.service import Service
+from ducktape.utils.util import wait_until
+from ducktape.cluster.cluster import ClusterNode
+
+from polaris.management.api_client import ApiClient
+from polaris.management.configuration import Configuration
+from polaris.management.api.polaris_default_api import PolarisDefaultApi
+
+
+class PolarisCatalog(Service):
+    """Polaris Catalog service
+    
+    The polaris catalog service maintain lifecycle of catalog process on the nodes. 
+    The service deploys polaris in a test mode with in-memory storage which is intended 
+    to be used for dev/test purposes.
+    """
+    PERSISTENT_ROOT = "/var/lib/polaris"
+    INSTALL_PATH = "/opt/polaris"
+    JAR = "polaris-service-1.0.0-all.jar"
+    JAR_PATH = os.path.join(INSTALL_PATH, "polaris-service/build/libs", JAR)
+    LOG_FILE = os.path.join(PERSISTENT_ROOT, "polaris.log")
+    POLARIS_CONFIG = os.path.join(PERSISTENT_ROOT, "polaris-server.yml")
+    logs = {
+        # Includes charts/ and results/ directories along with benchmark.log
+        "polaris_logs": {
+            "path": LOG_FILE,
+            "collect_default": True
+        },
+    }
+    # the only way to access polaris credentials running with the in-memory
+    # storage is to parse them from standard output
+    credentials_pattern = re.compile(
+        "realm: default-realm root principal credentials: (?P<client_id>.+):(?P<password>.+)"
+    )
+
+    nodes: list[ClusterNode]
+
+    def _java_path(self, node):
+        alternatives = node.account.ssh_capture("update-java-alternatives -l",
+                                                allow_fail=True)
+        for a in alternatives:
+            parts = a.split()
+            self.logger.debug(f"java alternatives: {parts}")
+            if 'java-1.21' in parts[0]:
+                return os.path.join(parts[2].strip(), "bin", "java")
+
```

Comment:
I think we can generate a wrapper script in the container with the right correct path and such.. since all of them are known at the image build.

Reply:
done